src/_pytest/pytester.py

```diff
@@ -687,11 +698,41 @@ def getinicfg(self, source):
         return py.iniconfig.IniConfig(p)["pytest"]
 
     def makepyfile(self, *args, **kwargs):
-        """Shortcut for .makefile() with a .py extension."""
+        r"""Shortcut for .makefile() with a .py extension
+        defaults to the test name .py e.g test_foobar.py
+        Calling without kwargs will result in overwriting the initial file contents
```

Comment:
I think we can remove this phrase, because it conveys the impression that by *calling* with kwargs initial file contents are somehow not overwritten.